mappings/net/minecraft/server/MinecraftServer.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/server/MinecraftServer
+	CLASS class_643 ServerThread
+	CLASS class_645 ConsoleInputThread
+	CLASS class_646 MagicalTimingHelperThread
+	FIELD field_2837 logger Ljava/util/logging/Logger;
+	FIELD field_2839 pendingConnectionManager Lnet/minecraft/class_9;
+	FIELD field_2840 serverProperties Lnet/minecraft/class_391;
+	FIELD field_2848 onlineMode Z
```

Comment:
how can you have an offline server :thinking:
This is before integrated servers

Reply:
online mode in server.properties determines if it should use the auth servers or not.
Open up a b1.7.3 server, you'll see you can turn it off

Reply:
should it be more specific like that? 🤔

Reply:
as in should the name be more specific

Reply:
I could change it to doAuthentication but that doesn't reflect the actual name for it in the server.properties or the "sane" name people would expect it to be.

Reply:
I'll wait for others to chime in before resolving this one way or the other.

Reply:
I think the names alright, it is the same as the properties and most server people know what that property is for.